packages/playwright-test/src/loader.ts

```diff
@@ -202,6 +202,10 @@ export class Loader {
     let snapshotDir = takeFirst(this._configOverrides.snapshotDir, projectConfig.snapshotDir, this._config.snapshotDir, testDir);
     if (!path.isAbsolute(snapshotDir))
       snapshotDir = path.resolve(configDir, snapshotDir);
+    const name = takeFirst(this._configOverrides.name, projectConfig.name, this._config.name, '');
+    let screenshotsDir = takeFirst(this._configOverrides.screenshotsDir, projectConfig.screenshotsDir, this._config.screenshotsDir, path.join(rootDir, '__screenshots__', process.platform, name));
```

Comment:
`path.join(rootDir, '__screenshots__', process.platform, name)` is something we never did before.
- `outputDir` uses `process.cwd()` as base;
- `snapshotDir` uses `testDir` as base.